WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -222,11 +222,16 @@ - (BlogDetailsSection *)generalSectionViewModel
                                                  }]];
 
     if ([Feature enabled:FeatureFlagPlans] && [self.blog supports:BlogFeaturePlans]) {
-        [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Plans", @"Action title. Noun. Links to a blog's Plans screen.")
-                                                        image:[UIImage imageNamed:@"icon-menu-plans"]
-                                                     callback:^{
-                                                         [weakSelf showPlans];
-                                                     }]];
+        BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Plans", @"Action title. Noun. Links to a blog's Plans screen.")
+                                                              image:[UIImage imageNamed:@"icon-menu-plans"]
+                                                           callback:^{
+                                                               [weakSelf showPlans];
+                                                           }];
+
+        Plan *plan = [[Plan alloc] initWithPlan:[self.blog.planID integerValue]];
```

Comment:
I wrote the ObjC bridge testing with a manually created `PlanEnum` value. If the compiler doesn't complain about this, it means it could be called with an invalid integer that doesn't represent any `PlanEnum` case. I'm not sure what happens then, but I have the feeling it'll crash. We should change `PlanObjc.init` to be a failable initializer.